testcases/TC14.1-all-dna-1.md

```diff
@@ -0,0 +1,53 @@
+---
+# The layout must be 'testcase'; DO NOT Change
+layout: testcase
+
+# Brief, descriptive title for the test case
+title: Link Purpose(In Context) - no links (TC14-1-all-dna-1)
```

Comment:
recheck the test case file name - should be TC14.1-ic-dna-1, or something like that

Reply:
Reviewed the Excel sheet and the test case name is correct as TC 14-1-all-dna-1? Should I change it to IC like the other DNAs???